modules/apps/users-admin/users-admin-impl/src/main/java/com/liferay/users/admin/internal/search/UserBulkReindexer.java

```diff
@@ -63,15 +118,9 @@ public void reindex(long companyId, Collection<Long> classPKs) {
 		}
 	}
 
-	@Reference(
-		target = "(indexer.class.name=com.liferay.portal.kernel.model.User)"
-	)
-	protected Indexer<User> indexer;
-
-	@Reference
-	protected UserLocalService userLocalService;
-
 	private static final Log _log = LogFactoryUtil.getLog(
 		UserBulkReindexer.class);
 
+	private int _databaseInMaxParameters = Integer.MAX_VALUE;
+
```

Comment:
@marianoalvarosaiz, let's define the var `_databaseInMaxParameters` for zero(`private int _databaseInMaxParameters;`)